raw/GB/language-tables/Komi_Zyrian.csv

```diff
@@ -1,196 +1,196 @@
+GB193,What is the order of adnominal property word and noun?,1,,,GB193
+GB196,Is there a masculine/feminine distinction in phonologically independent second person pronouns?,0,,,
+GB197,Is there a masculine/feminine distinction in phonologically independent first person pronouns?,0,,,
+GB198,Can an adnominal numeral agree with the noun in noun class/gender?,0,,,
+GB203,What is the order of the adnominal collective universal quantifier (‘all’) and the noun?,1,,"When nominalised with the definitive/possessive suffix and used in adposition, it can appear on either side of the described NP.",GB204-1
+GB204,Do collective (‘all’) and distributive (‘every’) universal quantifiers differ in their forms or their syntactic positions?,1,,,"GB204-1, GB204-2"
+GB250,Can predicative possession be expressed with a transitive ‘habeo’ verb?,0,,,
+GB252,Can predicative possession be expressed with an S-like possessum and a locative-coded possessor?,0,,,
+GB253,Can predicative possession be expressed with an S-like possessum and a dative-coded possessor?,0,,,
+GB254,Can predicative possession be expressed with an S-like possessum and a possessor that is coded like an adnominal possessor?,1,,A possessor is in GEN.,GB254
+GB256,Can predicative possession be expressed with a comitative/conjunctional construction?,0,,,
+GB257,Can polar interrogation be marked by intonation only?,1,SKJa 1967: 23-24,,
+GB260,Can polar interrogation be indicated by a special word order?,1,,,
+GB262,Is there a clause-initial polar interrogative particle?,0,,Russian-borrowed разьве and неужели are used word-initially to form polar questions.,
+GB263,Is there a clause-final polar interrogative particle?,0,,,
+GB264,Is there a polar interrogative particle that most commonly occurs neither clause-initially nor clause-finally?,1,,,GB264
+GB265,"Is there a comparative construction that includes a form that elsewhere means ‘surpass, exceed’?",0,,,
+GB266,Is there a comparative construction that employs a marker of the standard which elsewhere has a locational meaning?,1,,Elative is used for comparison.,GB266
+GB270,Can comparatives be expressed using two conjoined clauses?,0,,,
+GB273,Is there a comparative construction with a standard marker that elsewhere has neither a locational meaning nor a ‘surpass/exceed’ meaning?,1,,"Besides the elative case, postpositions дорысь and серти can be used to mark comparison, but the former also has locational meaning.",GB273
+GB275,Is there a bound comparative degree marker on the property word in a comparative construction?,1,,,GB266
+GB276,Is there a non-bound comparative degree marker modifying the property word in a comparative construction?,1,ÖKK 2000:510,The Russian-loaned particle бӧльӧ can be used in colloquial speech.,GB276
+GB285,Can polar interrogation be marked by a question particle and verbal morphology?,0,,,
+GB286,Can polar interrogation be indicated by overt verbal morphology only?,?,,,
```

Comment:
It seems the value has been changed to `?`. Isn't this PR only intended to change formatting?